AUTHORS

```diff
@@ -3021,5 +3021,106 @@ David Gilbertson <gilbertson.david@gmail.com>
 Sergey Zelenov <sergey.zelenov@getyourguide.com>
 Eric Bickle <ebickle@users.noreply.github.com>
 Joe Pea <joe@trusktr.io>
+ExE Boss <3889017+ExE-Boss@users.noreply.github.com>
+Mateusz Krawczuk <krawczukmat@gmail.com>
+Jonathan MERCIER <jmercier@cng.fr>
+Ujjwal Sharma <ryzokuken@igalia.com>
+Jichan <development@jc-lab.net>
+Hassaan Pasha <hassaan.pasha@teamo.io>
+Eric Dobbertin <eric@dairystatedesigns.com>
+Victor <doc999tor@gmail.com>
+Ling Samuel <lingsamuelgrace@gmail.com>
+Syohei YOSHIDA <syohex@gmail.com>
+Nitin Kumar <snitin315@gmail.com>
+Christian Niederer <aqpoba@gmail.com>
+Sk Sajidul Kadir <sheikh.sajid522@gmail.com>
+Bartlomiej Brzozowski <bart.brzozowski@janeasystems.com>
+Saajan <saajan.sn@gmail.com>
+Yash Ladha <yashladhapankajladha123@gmail.com>
+Xu Meng <dmabupt@gmail.com>
+Alex R <alexei.rudenko@gmail.com>
+Hachimi Aa (Sfeir) <hachimi.aa@sfeir.com>
+Daniel Estiven Rico Posada <danielrico.posada@gmail.com>
+Ruben Bridgewater <ruben.bridgewater@maibornwolff.de>
```

Comment:
Another duplicate

Reply:
Done 👍

Reply:
You also need to add this email address to the `.mailmap`, otherwise it will show up again during the next update